### PR TITLE
webkinz 1.7.5 (new cask)

### PR DESCRIPTION
**Pull request by octoturt**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---
It should be noted that this cask is for Webkinz Classic, _not_ Webkinz Next (often referred to simply as "Webkinz"), which is already available on the [Mac App Store](https://apps.apple.com/us/app/webkinz-family-pet-game/id1455137730).

If I've made any mistakes while writing this then feel free to let me know, I haven't gotten deep in the weeds writing formulae for Homebrew and this was a learning experience for me.

### CODE REVIEW

File: Casks/w/webkinz.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  desc "Virtual pet MMO"
+  homepage "https://webkinz.com/"
+
+  livecheck do
+    url "http://downloads.webkinz.com/apps/latest-mac.yml"
+    strategy :electron_builder
+  end
+
+  depends_on macos: ">= :el_capitan"
+  depends_on arch: :x86_64
```
Comment (bevanjkay) [inline Casks/w/webkinz.rb:16]: Can the software not be run on Apple Silicon with Rosetta?

**Comment by krehel** [comment]
How actively is this "classic" version developed?  You mentioned the "Next" version is on the App Store.  Is this expected to also move to MAS?

**Comment by octoturt** [comment]
> How actively is this "classic" version developed?

Classic has continued to receive new pets, items, and events on a very regular basis despite Next having been released years ago, and shows no sign of stopping. It remains the most popular version of the game and I don't expect them to pull it down anytime soon.

> You mentioned the "Next" version is on the App Store.  Is this expected to also move to MAS?

Next was designed from the ground up for both mobile and desktop and therefore shares a single listing. Classic has an [iOS version](https://apps.apple.com/us/app/webkinz-classic/id680610650), but it's different from the desktop version and probably doesn't translate so easily. This is all a long way of saying I doubt it because I don't see a reason for Ganz to do it.

**Comment by krehel** [comment]
Fair enough points.  Thank you!

n.b. We try to include most software provided it meets the criteria for inclusion, but generally try to avoid new additions which are not receiving updates or are likely to be moved fully to MAS (the latter has usually shown that non-MAS stuff gets abandoned)

File: Casks/w/webkinz.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  app "Webkinz.app"
+
+  zap trash: [
+    "~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/ganz.webkinz.sfl*",
+    "~/Library/Application Support/Webkinz",
+    "~/Library/Logs/Webkinz",
+    "~/Library/Saved Application State/Ganz.Webkinz.savedState",
+    "~/Preferences/Ganz.Webkinz.plist",
```
Comment (krehel) [inline Casks/w/webkinz.rb:25]: Should this be `~/Library/Preferences` ?

File: Casks/w/webkinz.rb (hunk 1 above)
Comment (krehel) [reply Casks/w/webkinz.rb:16]: Agreed, this should probably be dropped and replaced with the caveats block for requires_rosetta.

Comment (octoturt) [reply Casks/w/webkinz.rb:16]: I believe it can be. My bad! I must have misunderstood Homebrew's documentation.

Comment (krehel) [reply Casks/w/webkinz.rb:16]: No worries @octoturt!

`depends_on arch:` is for software that specifically requires the underlying architecture.

Normal Intel-built software that can use Rosetta should be placed in the caveats block before the final `end`.

```
caveats do
  requires_rosetta
end
```

File: Casks/w/webkinz.rb (hunk 2 above)
Comment (octoturt) [reply Casks/w/webkinz.rb:25]: It should! I'll fix that in a moment.

**Review by p-linnane — APPROVED** [review]
Thanks!

**Pull request merged**